### PR TITLE
download_libs downloader using wget2

### PR DESCRIPTION
**Pull request by dimitre**
So all files can be downloaded faster, in parallel
if wget2 is installed locally
if not it will fallback to curl, then wget
cc @danoli3

### CODE REVIEW

**Comment by danoli3** [comment]
Nice1!

**Comment by dimitre** [comment]
I think it is finally passing tests. I've simplified the functions to invoke the downloader, so multiple files can be passed easily. wget2 and curl download faster in parallel, wget download in series.

**Comment by ofTheo** [comment]
@dimitre if this is good to merge - looks good to me :)

File: scripts/dev/download_libs.sh [hunk 1]
```diff
@@ -9,278 +12,284 @@ BLEEDING_EDGE=0
...
 
+PARAMS=""
 for PKG in $PKGS; do
-    download $PKG
+	PARAMS+="https://github.com/openframeworks/apothecary/releases/download/$REPO/$PKG "
 done
+# echo " PARAMS $PARAMS"
+
+
+if command -v wget2 2>/dev/null; then
```
Comment (danoli3) [inline scripts/dev/download_libs.sh:230]: Should just do this in downloader 
https://github.com/openframeworks/openFrameworks/blob/master/scripts/dev/downloader.sh#L25

File: scripts/msys2/buildAllExamples.sh
Comment (danoli3) [inline scripts/msys2/buildAllExamples.sh:1]: do we need 755 
"755 means you can do anything with the file or directory, and other users can read and execute it but not alter it. Suitable for programs and directories you want to make publicly available. 644 means you can read and write and execute the file or directory and other users can only read it"

File: scripts/dev/download_libs.sh (hunk 1 above)
Comment (danoli3) [reply scripts/dev/download_libs.sh:230]: if wget2, else curl, else wget 
```
if command -v wget2 2>/dev/null; then 
        wget2 $@ 2> /dev/null;
elif command -v curl 2>/dev/null; then 
        curl -LO --retry 20 -O -s $@; 
else 
        wget -q $@ 2> /dev/null; 
fi
```

**Review by danoli3 — CHANGES_REQUESTED** [review]

File: scripts/msys2/buildAllExamples.sh
Comment (dimitre) [reply scripts/msys2/buildAllExamples.sh:1]: reverting this one. I've used chmod +x to execute on my system, to test some stuff on macOS from msys2 scripts

**Comment by dimitre** [comment]
Closing this in favor of https://github.com/openframeworks/openFrameworks/pull/8002

**Pull request closed without merging**